enzy_htp/_interface/pymol_interface.py

```diff
@@ -101,6 +102,104 @@ def select_pymol_obj(self, pattern: str, pymol_obj_name: str,
+                pymol_session.cmd.wizard("mutagenesis")
+                pymol_session.cmd.do("refresh_wizard")
+                pymol_session.cmd.get_wizard().set_mode(target)
+                if pymol_obj_name == "":
+                    pymol_session.cmd.get_wizard().do_select(f"{pos_key[0]}/{str(pos_key[1])}/")
+                else:
+                    pymol_session.cmd.get_wizard().do_select(f"{pymol_obj_name}//{pos_key[0]}/{str(pos_key[1])}/")
+            
+                pymol_session.cmd.get_wizard().do_state(i)
+                pymol_session.cmd.frame(i)
+                pymol_session.cmd.get_wizard().apply()
+                self.export_pdb(pymol_obj_name, pymol_session, tag="rotamer_" + str(i))
+            return
+
+        pymol_session.cmd.get_wizard().apply()
+
+    def export_pdb(self, pymol_obj_name: str, 
+                        pymol_session: pymol2.PyMOL,
+                        if_retain_order: bool = True,
+                        tag: str = None) -> str:
+        """
+        Saves a PyMOL object to a PDB file.
+        Args:
+            pymol_obj_name: the name of the target enzyme in PyMOL.
+            pymol_session: the target PyMOL session.
+            in_order: if the saving should keep the order of the atoms in the original object.
+            tag: the name tag for the saved file.
+        Returns:
+            The path to the saved PDB file.
+        """
+        result_dir = eh_config["system.SCRATCH_DIR"]
+        fs.safe_mkdir(result_dir)
+        if tag is not None:
+            pymol_outfile_path = fs.get_valid_temp_name(
+                    f"{result_dir}/{pymol_obj_name}_{tag}.pdb")
+        else:
+            pymol_outfile_path = fs.get_valid_temp_name(
+                f"{result_dir}/{pymol_obj_name}.pdb")
+
+        if if_retain_order:
+            pymol_session.cmd.set("retain_order")
+        pymol_session.cmd.save(pymol_outfile_path, pymol_obj_name)
+
+        return pymol_outfile_path
+
+
+    def export_enzy_htp_stru(self, pymol_obj_name: str,
+                            pymol_session: pymol2.PyMOL,
+                            if_retain_order: bool = True,
+                            if_multichain: bool = False) -> Structure:
+        """
+        Saves a PyMOL object to a Structure object.
+        Args:
+            pymol_obj_name: the name of the target enzyme in PyMOL.
+            pymol_session: the target PyMOL session.
+            in_order: if the saving should keep the order of the atoms in the original object.
+            if_multichain: if the Structure object created will have more than one chain.
+        Returns:
+            A Structure object representing the target enzyme in PyMOL.
+        """
+        sp = PDBParser()
+        pymol_outfile_path = self.export_pdb(pymol_obj_name, pymol_session, if_retain_order=if_retain_order)
+        res = sp.get_structure(pymol_outfile_path, allow_multichain_in_atom=if_multichain)
```

Comment:
I think this option can always be True in this function. Since the caller of this function will not necessarily know if the object have multichain. Let just hard code it as True here and remove the if_multichain and change when problems show up.